subprojects/ce-login/cli/CliCreateHsf.cpp

```diff
@@ -393,110 +268,13 @@ CeLogin::CeLoginRc cli::createHsf(int argc, char** argv)
-                {
-                    cout << "Wrote: " << sArgs.mJsonHashPath << endl;
-                }
-                else
-                {
-                    cout << "Error in file" << endl;
-                }
-            }
-        }
-        if (sMode == Mode_Sign)
-        {
-            std::vector<uint8_t> sJsonDigest;
-            std::vector<uint8_t> sSignature;
-            bool sIsDigestValid =
-                readBinaryFile(sArgs.mJsonHashPath, sJsonDigest);
-
-            if (sIsDigestValid)
-            {
-                sRc = CeLogin::createCeLoginAcfV1Signature(
-                    sCreateHsfArgs, sJsonDigest, sSignature);
-            }
-            else
-            {
-                cout << "Unable to read digest file" << endl;
-            }
-
-            if (writeBinaryFile(sArgs.mSignaturePath,
-                                (const uint8_t*)sSignature.data(),
-                                sSignature.size()))
-            {
-                cout << "Wrote: " << sArgs.mSignaturePath << endl;
-            }
-            else
-            {
-                cout << "Error in file" << endl;
-            }
-        }
-        if (sMode == Mode_Asn1)
-        {
-            std::vector<uint8_t> sJson;
-            std::vector<uint8_t> sSignature;
-            std::vector<uint8_t> sAcf;
-            bool sIsJsonValid = readBinaryFile(sArgs.mJsonPath, sJson);
-            bool sIsSignatureValid =
-                readBinaryFile(sArgs.mSignaturePath, sSignature);
-
-            if (sIsJsonValid && sIsSignatureValid)
-            {
-                string sJsonStr = string((char*)sJson.data(),
-                                         (char*)sJson.data() + sJson.size());
-                sRc = CeLogin::createCeLoginAcfV1Asn1(sCreateHsfArgs, sJsonStr,
-                                                      sSignature, sAcf);
-            }
-            else
-            {
-                cout << "Unable to read digest file" << endl;
-            }
+        vector<uint8_t> sAcfBinary;
+        sRc = CeLogin::createCeLoginAcfV1(sCreateHsfArgs, sAcfBinary);
 
-            if (writeBinaryFile(sArgs.mOutputFile, (const uint8_t*)sAcf.data(),
-                                sAcf.size()))
-            {
-                cout << "Wrote: " << sArgs.mOutputFile << endl;
-            }
-            else
-            {
-                cout << "Error in file" << endl;
-            }
-        }
-        else if (sMode == Mode_Full)
+        if (!writeBinaryFile(sArgs.mOutputFile, sAcfBinary.data(),
+                             sAcfBinary.size()))
         {
-            vector<uint8_t> sAcfBinary;
-            sRc = CeLogin::createCeLoginAcfV1(sCreateHsfArgs, sAcfBinary);
-
-            if (!writeBinaryFile(sArgs.mOutputFile, sAcfBinary.data(),
-                                 sAcfBinary.size()))
-            {
-                cout << "Error in file" << endl;
-            }
+            cout << "Error in file" << endl;
```

Comment:
Please  clarify this error is writing a file, perhaps: "Error writing ACF file" + sArgs.mOutputFile
I see this error when the directory is not writable, and a better error will help users understand how to fix the problem.